srcpkgs/xournalpp/template

```diff
@@ -11,8 +11,9 @@ short_desc="Handwriting Notetaking software with PDF annotation support"
 maintainer="mobinmob <mobinmob@disroot.org>"
 license="GPL-2.0-or-later"
 homepage="https://github.com/xournalpp/xournalpp"
+changelog="https://raw.githubusercontent.com/xournalpp/xournalpp/1.0.20/CHANGELOG.md"
```

Comment:
Using the changelog from master is better, since it can show the differences both for the current version as well as new versions when updating.

Reply:
Done, thanks for the review.